lib/internal/assert.js

```diff
@@ -10,13 +10,18 @@ let green = '';
 let red = '';
 let white = '';
 
-const READABLE_OPERATOR = {
-  deepStrictEqual: 'Input A expected to strictly deep-equal input B',
-  notDeepStrictEqual: 'Input A expected to strictly not deep-equal input B',
-  strictEqual: 'Input A expected to strictly equal input B',
-  notStrictEqual: 'Input A expected to strictly not equal input B'
+const kReadableOperator = {
+  deepStrictEqual: 'Expected input to be strictly deep-equal',
+  notDeepStrictEqual: 'Expected "actual" not to be strictly deep-equal',
```

Comment:
why use "actual" here?

Reply:
It is the value of the input argument "actual". I had hoped that this is actually clearer than the version before. Do you have a idea or a suggestion for to make it even better?

Reply:
I'm wondering why it's not `input` or `inputs` like with the sibling method

Reply:
I see. Thanks for the explanation, this seems fine.